docs/installation.rst

```diff
@@ -236,10 +236,10 @@ Install the dependencies automatically using zypper::
 Alternatively, you can enumerate all build dependencies in the command line::
 
     sudo zypper install python3 python3-devel \
-    libacl-devel openssl-devel libxxhash-devel \
+    libacl-devel openssl-devel xxhash-devel \
     python3-Cython python3-Sphinx python3-msgpack-python python3-pkgconfig pkgconf \
     python3-pytest python3-setuptools python3-setuptools_scm \
-    python3-sphinx_rtd_theme gcc gcc-c++
+    python3-sphinx_rtd_theme gcc gcc-c++ libzstd-devel liblz4-devel
```

Comment:
maybe rather move these compression libs/headers directly behind the xxhash-devel. borg directly depends on these, so they belong rather there.

Reply:
Fixed

Reply:
that was quick! :-)